docker-compose.yml

```diff
@@ -143,7 +143,7 @@ services:
     image: fleetdm/docker-idp:latest
     environment:
       SIMPLESAMLPHP_SP_ENTITY_ID: "https://localhost:8080"
-      SIMPLESAMLPHP_SP_ASSERTION_CONSUMER_SERVICE: "https://localhost:8080/api/latest/fleet/sso/callback"
+      SIMPLESAMLPHP_SP_ASSERTION_CONSUMER_SERVICE: "https://localhost:8080/api/v1/fleet/sso/callback"
```

Comment:
This is why the tests passed and we didn't notice.

Reply:
Ah I see, so if we introduced a mismatch here it'd fail the validation in the callback, even if the URL is handled properly.

Reply:
Ah, it probably doesn't care the path matches (it's a "Testing SAML"). We must test with a proper IdP.